codesy/templates/includes/get_started.html

```diff
@@ -39,12 +39,29 @@
         <div id="payment-errors"></div>
 
         {% if request.user.stripe_customer%}
-            <h3>You are ready to bid</h3>
-            <p>Try checking out GitHub issues with the most +1 votes.</p>
-            <a class="button gh-button track-btn" data-category="top+1" data-label="Created by You" href="https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+sort%3Areactions-%2B1-desc+author%3A{{ request.user.username }}"><i class="fi-social fi-social-github medium"></i> Top +1 Issues Created by You</a>
-            <a class="button gh-button track-btn" data-category="top+1" data-label="Assigned to You" href="https://github.com/issues?q=is%3Aopen+is%3Aissue+sort%3Areactions-%2B1-desc+assignee%3A{{ request.user.username }}"><i class="fi-social fi-social-github medium"></i> Top +1 Issues Assigned to You</a>
-            <a class="button gh-button track-btn" data-category="top+1" data-label="Mentioning You" href="https://github.com/issues?q=is%3Aopen+is%3Aissue+sort%3Areactions-%2B1-desc+mentions%3A{{ request.user.username }}"><i class="fi-social fi-social-github medium"></i> Top +1 Issues Mentioning You</a><br/>
-            <a class="button gh-button track-btn" data-category="top+1" data-label="Global" href="https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+sort%3Areactions-%2B1-desc+"><i class="fi-social fi-social-github medium"></i> Global Top +1 Issues</a><br/>
+            <h4 class="text-left" >You are ready to start bidding.  You can can bid on your own github issues or check out some of the latest activity on codesy:</h4>
```

Comment:
I think we should keep a "You are ready to bid" header separate from other instructions. Removing the buttons will help draw attention to it, but we want to make it clear to the new user that they are done - they can just use GitHub as normal and codesy can take care of the rest.

An activity table is cool, but let's set it apart from the "You are ready to start bidding. [Go use GitHub like normal.]" header.